src/xo-app/logs/index.js

```diff
@@ -102,23 +104,30 @@ const JobDataInfos = ({
 )
 
 const CALL_FILTER_OPTIONS = [
-  { label: 'successfulJobCall', value: 'success' },
+  { label: 'allJobCalls', value: 'all' },
   { label: 'failedJobCall', value: 'error' },
   { label: 'jobCallInProgess', value: 'running' },
-  { label: 'allJobCalls', value: 'all' },
+  { label: 'jobCallSkipped', value: 'skipped' },
+  { label: 'successfulJobCall', value: 'success' },
 ]
 
 const PREDICATES = {
   all: () => true,
-  error: call => call.error !== undefined,
+  skipped: call => call.error !== undefined && !isAnError(call.error),
+  error: call => call.error !== undefined && isAnError(call.error),
   running: call => call.end === undefined && call.error === undefined,
   success: call => call.end !== undefined && call.error === undefined,
 }
 
 const UNHEALTHY_VDI_CHAIN_ERROR = 'unhealthy VDI chain'
+const NO_SUCH_OBJECT_ERROR = 'no such object'
 const UNHEALTHY_VDI_CHAIN_LINK =
   'https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection'
 
+const isAnError = error =>
```

Comment:
IMHO, the name does not reflect enough what this function does.

Maybe reverse the test and name it something like `isSkippedError`?